### PR TITLE
Adds a custom Sharesight Top 20 blog category.

### PR DESCRIPTION
**Pull request by kylorhall**
[ticket](https://vimaly.com/#j8mqm/t/www_Give_the_Sharesight_Top_20_a_home_with_a_Speci.../nOk1G95pxewBLkfN6)

As of opening, needs a bit of content to finish it off.

### CODE REVIEW

File: source/blog/paginated.html.erb [hunk 1]
```diff
@@ -91,6 +109,13 @@ include_campaign_monitor: true
       </aside>
     </div>
 
-    <%=partial 'blog/partials/blog_menu', locals: {locale_id: locale_obj[:id], category_name: category[:name], footer: true, on_individual_page: false} %>
+    <% if show_menu %>
```
Comment (alto) [inline source/blog/paginated.html.erb]: I like this semantic. Much better than `if !is_top_20_category` 👍

File: source/blog/paginated.html.erb [hunk 2]
```diff
@@ -4,18 +4,36 @@ section_class: 'blog_paginated section_blog'
...
+        <a href="<%= unlocalized_url('blog') %>" title="Sharesight Blog"><h1 class="heading_page">Blog</h1></a>
+      <% elsif is_top_20_category %>
+        <h1 class="heading_page"><%= space_category_page_title %></h1>
 
-      <% if category[:name] != 'All' %>
-        <h1 class="heading heading_page heading_page_index">Topic: <%= space_category_page_title %></h1>
+        <div class="campaign_monitor">
+          <%#
+            Campaign Monitor "Subscribe" Button.
+            TODO: This should be specifically for the Sharesight Top 20
```
Comment (alto) [inline source/blog/paginated.html.erb]: What's the `TODO` for? Isn't it now already Sharesight Top 20 only?

Comment (kylorhall) [reply source/blog/paginated.html.erb]: No, need content from marketing.

Comment (alto) [reply source/blog/paginated.html.erb]: I see.

**Pull request merged**